cleanslate/cleanslate/urls.py

```diff
@@ -29,6 +29,7 @@
     url(r'^doma/reminders/$', reminders, name='reminder'),
     url(r'^doma/finance/$', finance, name='finance'),
     url(r'^doma/calendar/$', calendar, name='calendar'),
+    url(r'^doma/message-board/$', forum, name='message-board'),
```

Comment:
Why did you go for message-board instead of forum?

Reply:
Just personal preference, it felt more message-boardy than forumy to me